src/main/java/step4/domain/lotto/Lotto.java

```diff
@@ -0,0 +1,33 @@
+package step4.domain.lotto;
+
+import java.util.Arrays;
+import java.util.Set;
+import java.util.TreeSet;
+import java.util.stream.Collectors;
+
+/*
+ * Lotto
+ * ver. 1.0
+ * 2020.05.31
+ */
+public class Lotto {
+
+    // each Lotto numbers.
+    private Set<Integer> lottoNumbers;
+
+    public Lotto(TreeSet<Integer> lottoNumbers) {
+        LottoNumber.checkLottoRules(lottoNumbers);
+        this.lottoNumbers = lottoNumbers;
+    }
+
+    public Lotto(String lottoNumbers) {
+        this.lottoNumbers = new TreeSet<>(Arrays.asList(lottoNumbers.split(LottoNumber.FIXED_DELIMITER)).stream()
+                .map(Integer::parseInt)
+                .collect(Collectors.toSet()));
```

Comment:
```suggestion
       this.lottoNumbers = Arrays.asList(lottoNumbers.split(LottoNumber.FIXED_DELIMITER)).stream()
                .map(Integer::parseInt)
                .collect(Collectors.collectingAndThen(Collectors.toSet(), TreeSet::new));
```
collectingAndThen을 사용해보세요 😄

Reply:
무슨 목적의 함수인가 추가 학습했네요 ^^ 감사합니다.